spec/javascripts/search-spec.js

```diff
@@ -0,0 +1,22 @@
+describe('Search', function() {
+  'use strict';
+
+  var module;
+  var content;
+  var query;
+  var processedContent;
+  var expectedResults;
+
+  beforeEach(function() {
+    module = new GOVUK.Modules.Search();
+    processedContent = module.processContent(content, query);
+  });
+
+  it('returns concatenated sentences that contain the search query', function() {
+    expect(processedContent).toEqual(expectedResults)
```

Comment:
I'm was confused by this because `expectedResults` is used before it's set. Would it be better to move everything into the `it` block? I'm still not entirely sure we're not accidentally doing `expect(null).toEqual(null)` here.